web/site/components/Sbc/NuxtContentCard.vue

```diff
@@ -18,11 +18,16 @@ const props = defineProps({
 
 const fullId = 'content-data-' + props.id
 
-const { data } = await useAsyncData(fullId, () => {
+// Fetch the content data based on the current locale and route
+const { data, refresh } = await useAsyncData(fullId, () => {
```

Comment:
Can you not use the watch option provided by useAsyncData?

https://nuxt.com/docs/api/composables/use-async-data#watch-params

Reply:
Sure, just updated